test/README.md

```diff
@@ -1,7 +1,29 @@
+This folder contains testing infrastructure for the pipelines in the skylab repository.
+
+**Description of Tests**
```

Comment:
Add a brief description (edit the example below for accuracy):
"When a pull request (PR) is made for the Skylab Rep, the following tests are triggered automatically, except for tests marked as manual. " These tests will need to be run using the instructions in the 'Running tests manually' section."

Reply:
Change to:
## Description of tests and optional test variants